Documentation/security/policy/language.rst

```diff
@@ -29,8 +29,8 @@ can talk to each other. Layer 3 policies can be specified using the following me
   to the local host serving the endpoints or all connectivity to outside of
   the cluster.
 
-* `Node based`: This is an extension of ``remote-node`` entity. Optionally nodes
-   can have unique identity that can be used to allow/block access only from specific ones.
+* `Node based`: This is an extension of `remote-node` entity. Optionally nodes 
```

Comment:
Ah, sorry, I misread what the change was here. The double grave (``` `` ```) means that the text should be treated as a verbatim term, whereas the single grave (``` ` ```) creates a hyperlink. Since there is no sphinx anchor for the term `remote-node`, this causes the docs build to fail:

https://github.com/cilium/cilium/actions/runs/14579803104/job/40893808560?pr=39065#step:4:1261

```suggestion
* `Node based`: This is an extension of ``remote-node`` entity. Optionally nodes 
```

I can see that this sentence is malformed in the final render, and I'm not sure what the best way to solve that is. But this is not the solution.

Reply:
Ah, sorry about that, i should've checked this. Fixed here: https://github.com/cilium/cilium/commit/1e26e8b5c4216419622eadc07532990377848fc2

Reply:
AFAICT if i render the docs based of this branch the sentence is fixed.

Reply:
I suspect what happens is that by changing it to a hyperlink, the hyperlink can be rendered correctly in the theme. That way it doesn't apply bold formatting when it's part of an unordered list like this line is. This fixes the rendering to look right. But then the hyperlink is a dead link because there is nothing to link to. This is what then causes the build failure in CI. There's probably a way to fix this in the theme.

Reply:
Maybe the simplest solution to this problem is to come up with different phrasing for the feature so that it doesn't directly reference the `remote-node` entity and instead talks more generally about how node selectors can be used to select traffic to or from nodes within the cluster and apply policy based on metadata associated with the node.